sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -13,7 +13,6 @@
 from .blob_service_client import BlobServiceClient
 from .lease import LeaseClient
 from ._shared.policies import ExponentialRetry, LinearRetry, NoRetry
-from ._shared.downloads import StorageStreamDownloader
```

Comment:
Maybe we want to expose it to user I will add it back in the next commit

Reply:
I think this probably should be exposed - it is the return type of the `download` operations and if users what to do Type-checking they will need to be able to import this type.
I also think it it's exposed in the Files SDK, we whichever way we go should be consistent.

Reply:
It wasn't exposed in File I think, but I assumed we want to expose it